### PR TITLE
Add data to store.config file

### PR DESCRIPTION
**Pull request by veredrec**
Relates #20 

Add data for 5 jobs with: 

- title
- tagline
- description
- degree
- image
- degree
- grade_bagrut
- grade_psychometric
- universities
- salary_start
- salary_ten_year

### CODE REVIEW

File: src/store.config.js [hunk 1]
```diff
@@ -4,16 +4,61 @@ import reducers from './reducers/index';
...
+      { title: 'Doctor',
+        tagline: 'Promots and Improves Health',
+        description: 'A doctor is a professional who practises medicine, which is concerned with promoting, maintaining, or restoring health.',
+        image: 'doctor.svg',
+        degree: 'Medicine',
+        grade_bagrut: '11.10',
+        grade_psychometric: '760',
+        universities: 'Hebrew University, Technion, Ben-Gurion University, Tel-Aviv University',
+        salary_start: '9,000',
+        salary_ten_year: '35,000'
```
Comment (matthewdking) [inline src/store.config.js]: Is this a yearly or monthly salary? And is it Shekels or dollars?

File: src/store.config.js [hunk 2]
```diff
@@ -4,16 +4,61 @@ import reducers from './reducers/index';
...
   const initialStore = {
     careers: [
-      { title: 'doctor',
-        tagline: 'saves lives',
-        description: 'this is the amazing doctor who is just really rich!',
-        image: 'doctor.svg'
+      { title: 'Doctor',
+        tagline: 'Promots and Improves Health',
+        description: 'A doctor is a professional who practises medicine, which is concerned with promoting, maintaining, or restoring health.',
+        image: 'doctor.svg',
```
Comment (matthewdking) [inline src/store.config.js]: I'm guessing we don't have these images yet?

Comment (veredrec) [reply src/store.config.js]: Nope, we should decide on what type of images we want to include.

File: src/store.config.js (hunk 1 above)
Comment (veredrec) [reply src/store.config.js]: It's monthly, in shekels. Want to move to Israel? :stuck_out_tongue_winking_eye:

**Comment by veredrec** [comment]
@sajedazoabi I think we said that colleges would be a stretch goal, just because there are so many is Israel...

File: src/store.config.js (hunk 1 above)
Comment (matthewdking) [reply src/store.config.js]: @veredrec I'm only staying if you are

**Pull request merged**